components/automate-cli/pkg/verifyserver/services/externalpostgresqlservice/externalpostgresqlconstants.go

```diff
@@ -0,0 +1,7 @@
+package externalpostgresqlservice
+
+const ExternalPgSuccessConnectionTitle  = "Connection successfully tested"
+const ExternalPgFailConnectionTitle     = "External Postgresql Connection failed"
+const ExternalPgConnectionErrorMsg      = "Machine is unable to connect with External Managed Postgresql"
+const ExternalPgConnectionResolutionMsg = "Ensure that the Postgres configuration provided is correct. Review security group or firewall settings as well on the infrastructure"
+const ExternalPgConnectionSuccessMsg    = "Connection successfully tested"
```

Comment:
Change the message to "Machine is able to connect with External Managed Postgres" as mentioned in swagger

Reply:
done